src/marker_group.js

```diff
@@ -15,8 +16,12 @@ Potential improvements:
 class MarkerGroup {
     /**
      * @param {EditSession} session
+     * @param {{markerType: "fullLine" | "line" | undefined}} [options]
```

Comment:
could you describe here what each argument does?

Reply:
added a JSDoc comment